packages/@react-spectrum/contextualhelp/src/ContextualHelp.tsx

```diff
@@ -46,7 +46,7 @@ function ContextualHelp(props: SpectrumContextualHelpProps, ref: FocusableRef<HT
   return (
     <DialogTrigger {...otherProps} type="popover" placement={placement} hideArrow>
       <ActionButton
-        {...mergeProps(otherProps, labelProps)}
+        {...mergeProps(otherProps, labelProps, {isDisabled: false})}
```

Comment:
user props win over provider props (what form uses to propagate isDisabled)
however, slot props win over user props

if we want to prevent people from setting isDisabled on ContextualHelp, we should remove it from the prop types that ContextualHelp accepts, and we should also wrap this ActionButton here in a Slots Provider that just has
```
            <SlotProvider
              slots={{
                actionButton: {
                  isDisabled: false
                }
              }}>
```
This will get merged with other slot props, and since this is the lowest one, it'll win for isDisabled.

Reply:
Do we care about slots? We don't publicly document them...

Reply:
yeah, but it'll prevent us from making this mistake ourselves in the future 🤣

Reply:
a whole extra provider just seems annoying/excessive though 😂